wee_slack.py

```diff
@@ -1151,6 +1151,11 @@ def update_member_presence(self, user, presence):
             if user.id in c.members:
                 c.update_nicklist(user.id)
 
+    def subscribe_users_presence(self):
+        self.send_to_websocket({
+            "type": "presence_sub",
+            "ids": self.users.keys(),
+            }, expect_reply=True)
```

Comment:
Do we expect a reply?

Reply:
Hmm no, I probably put it to true to make some tests and forgot to revert